### PR TITLE
chore: release v4.47.0

### PR DESCRIPTION
**Pull request by sarahdayan**
This pull request prepares the following release:
| Repository | Branch | Update | Change |
|---|---|---|---|
| [instantsearch.js](https://github.com/algolia/instantsearch.js) | releases/v4.47.0 | minor | [`4.46.3` → `4.47.0`](https://github.com/algolia/instantsearch.js/compare/v4.46.3...releases/v4.47.0) |

### Release Summary
This is going to be published with the following command:

```yarn publish --no-git-tag-version --non-interactive --tag latest```

### Merging Instructions
When merging this pull request, you need to **Squash and merge** and make sure that the title starts with `chore: release v4.47.0`.
<details>
<summary>See details</summary>

After that, a commit `chore: release v4.47.0` will be added and you or your CI can run `shipjs trigger` to trigger the release based on the commit.
Squash and merge
</details>

---

_This pull request is automatically generated by [Ship.js](https://github.com/algolia/shipjs)_.

### CODE REVIEW

**Comment by sarahdayan** [comment]
@Haroenv @dhayab  We can wait until your PRs are merged and you rebase them here to merge this one.

**Comment by Haroenv** [comment]
could we include https://github.com/algolia/instantsearch.js/pull/5127 @sarahdayan ?

**Review by Haroenv — CHANGES_REQUESTED** [review]
no merge to include the fix & class name

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+
+### Bug Fixes
+
+* **hierarchicalMenu:** pass correct attribute name to Insights ([#5124](https://github.com/algolia/instantsearch.js/issues/5124)) ([fe18a16](https://github.com/algolia/instantsearch.js/commit/fe18a168b1b195d067298770b55fd29a7fdb6edb))
+
+
+### Features
+
+* **status:** introduce status in InstantSearch class ([#5115](https://github.com/algolia/instantsearch.js/issues/5115)) ([21f3147](https://github.com/algolia/instantsearch.js/commit/21f31476e75e162b38b002d5439f231f3990e785))
+
```
Comment (Haroenv) [inline CHANGELOG.md:12]:
```suggestion
* **hierarchicalMenu**: introduce `ais-HierarchicalMenu-item--selected` class ([#5125](https://github.com/algolia/instantsearch.js/issues/5125)) ([4ebb828](https://github.com/algolia/instantsearch.js/commit/4ebb828c93afabfd8083246dfe7edfd33932d5fd))
```

**Review by FabienMotte — APPROVED** [review]

**Pull request merged**